py/desispec/test/test_binscripts.py

```diff
@@ -353,13 +353,6 @@ def test_compute_sky(self):
             inputs=inputs, outputs=outputs, clobber=True)
         self.assertTrue(success)
 
-def test_suite():
-    """Allows testing of only this module with the command::
-
-        python setup.py test -m <modulename>
-    """
-    return unittest.defaultTestLoader.loadTestsFromName(__name__)
-
 
 #- This runs all test* functions in any TestCase class in this file
 if __name__ == '__main__':
```

Comment:
These sections are also deprecated and should be removed.  The `pytest` equivalent is just:
```
pytest py/desispec/test/test_binscripts.py
```

Reply:
Done in https://github.com/desihub/desispec/pull/2155/commits/b4c874dfe82d61b91da9ec5f232d96fee39d4092 but please review that I didn't miss anything or over-deleted things.